internal/ent/enums/role.go

```diff
@@ -9,16 +9,17 @@ import (
 type Role string
 
 var (
-	RoleOwner  Role = "OWNER"
-	RoleAdmin  Role = "ADMIN"
-	RoleMember Role = "MEMBER"
-	Invalid    Role = "INVALID"
+	RoleOwner   Role = "OWNER"
+	RoleAdmin   Role = "ADMIN"
+	RoleMember  Role = "MEMBER"
+	RoleUser    Role = "USER"
+	RoleInvalid Role = "INVALID"
 )
 
 // Values returns a slice of strings that represents all the possible values of the Role enum.
-// Possible default values are "ADMIN", and "MEMBER".
+// Possible default values are "ADMIN", "MEMBER", "OWNER"
 func (Role) Values() (kinds []string) {
-	for _, s := range []Role{RoleAdmin, RoleMember} {
+	for _, s := range []Role{RoleAdmin, RoleMember, RoleOwner, RoleUser} {
```

Comment:
Owner is not always a valid role, its added to the values for organizations, but we don't have that as a valid role in groups which is why its not set in the default here. 

```suggestion
	for _, s := range []Role{RoleAdmin, RoleMember, RoleUser} {
```